requirements-build.txt

```diff
@@ -12,6 +12,7 @@ flask-swagger-ui
 Flask-WTF
 gitpython
 gunicorn
+lxml
```

Comment:
Oh, I think this should also go into `requirements-test.txt` -- commit upcoming.